src/br/org/scadabr/view/component/LinkComponent.java

```diff
@@ -44,17 +44,19 @@ public ImplDefinition definition() {
 	}
 
 	private void createLink() {
-		String content = createLinkContent();
-		setContent(content);
+		setContent(createLinkContent());
```

Comment:
What is the reason for this change? 

I would go back to how it was, if there is no need to change something, then we probably won't touch it.